### ISSUE TITLE
Zigbee Sonoff ZBBridge is dead?

### ISSUE THREAD

**Issue by abalashov78** [arendst/Tasmota#9383]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

I made like on this page https://www.digiblur.com/2020/07/how-to-use-sonoff-zigbee-bridge-with.html?m=1
with both latest tasmota-zbbridge.bin  release and developer versions
but I can't pair any devices with Sonoff ZBBridge with Tasmota FW.
I tested with both file ncp-uart-sw_6.7.6_115200.ota  or ncp-uart-sw_6.5.5_115200.ota 

in log i have only this message 

2020-09-25 08:37:30 DEBUG (bellows.thread_0) [bellows.uart] Sending: b'665721575485ea4da66bb5da55aac820fb0c42ceaca20be29ca800eccc47fb2c7e'
2020-09-25 08:37:30 DEBUG (bellows.thread_0) [bellows.uart] Data frame: b'6757a15754851563917e'
2020-09-25 08:37:30 DEBUG (bellows.thread_0) [bellows.uart] Sending: b'87009f7e'
2020-09-25 08:37:30 DEBUG (MainThread) [bellows.ezsp.protocol] Application frame 175 (addTransientLinkKey) received: b'00'

in tasmota console:
06:28:10 TCP: from MCU: 5646A1575479158BF57E
06:28:10 TCP: to MCU/1: 8610BE7E6647215754790FA25954927E
06:28:10 TCP: from MCU: 6747A1575479152FB97E
06:28:11 TCP: to MCU/1: 87009F7E
06:28:11 TCP: to MCU/1: 7744215754797D33A25912387E
06:28:11 TCP: from MCU: 7044A157547915CF667E
06:28:11 TCP: to MCU/1: 8070787E
06:28:11 TCP: to MCU/1: 0045215754797D38B75943EC7E
06:28:11 TCP: from MCU: 0145A157
06:28:11 TCP: from MCU: 547915B6C67E
06:28:11 TCP: to MCU/1: 8160597E
06:28:11 TCP: to MCU/1: 7D314A2157547937B0597D31937E
06:28:11 CFG: Saved to flash at F6, Count 16, Bytes 4096
06:28:11 TCP: from MCU: 124AA1575479159C3C7E
06:28:11 TCP: to MCU/1: 82503A7E224B2157547910A259E9417E
06:28:12 TCP: from MCU: 234BA1575479
06:28:12 TCP: from MCU: 1538707E
06:28:12 TCP: to MCU/1: 83401B7E
06:28:12 TCP: to MCU/1: 3348215754790CB059CAFA7E
06:28:12 TCP: from MCU: 3448A157547915D8
06:28:12 TCP: from MCU: AF7E
06:28:12 TCP: to MCU/1: 8430FC7E
06:28:12 TCP: to MCU/1: 44492157547919B0593FCA7E
06:28:12 TCP: from MCU: 4549A157547915A10F7E
06:28:12 TCP: to MCU/1: 8520DD7E
06:28:12 TCP: to MCU/1: 554E2157547907B2478DF07E
06:28:12 TCP: from MCU: 564EA15754791586B77E
06:28:12 TCP: to MCU/1: 8610BE7E
06:28:12 TCP: to MCU/1: 664F215754793FB159D9CD7E
06:28:12 TCP: from MCU: 674FA1
06:28:12 TCP: from MCU: 5754791522FB7E
06:28:13 TCP: to MCU/1: 87009F7E
06:28:13 TCP: to MCU/1: 774C21575479144D5989FE7E
06:28:13 TCP: from MCU: 704CA157547915
06:28:13 TCP: from MCU: C2247E
06:28:13 TCP: to MCU/1: 8070787E
06:28:13 TCP: to MCU/1: 004D215754781622677E
06:28:13 TCP: from MCU: 014DA1575478
06:28:13 TCP: from MCU: 15B85939377E
06:28:13 TCP: to MCU/1: 8160597E
06:28:13 TCP: to MCU/1: 7D31522157542814B6587BF425AA54924C3E0B7E
06:28:13 TCP: from MCU: 1252A157542815B6747E
06:28:13 TCP: to MCU/1: 82503A7E
06:28:13 TCP: to MCU/1: 2253215754211422307E
06:28:13 TCP: from MCU: 2353A157542105
06:28:13 TCP: from MCU: 4DA66BB5DA55AA6DB663B1D8541231986B317E
06:28:14 TCP: to MCU/1: 83401B7E
06:28:14 TCP: to MCU/1: 3350215754211753A97E
06:28:14 TCP: from MCU: 3450A1575421054D
06:28:14 TCP: from MCU: A66BB5DA55AA6DB663B1D854123198C63F7E
06:28:14 TCP: to MCU/1: 8430FC7E
06:28:14 TCP: to MCU/1: 4451215754800487317E
06:28:14 TCP: from MCU: 4551A157548015B5E9954C20AF55384F2C7E
06:28:14 TCP: to MCU/1: 8520DD7E
06:28:14 TCP: to MCU/1: 55562157543D5A7D337E
06:28:14 TCP: from MCU: 5656A157543D86
06:28:14 TCP: from MCU: E3A37E
06:28:14 TCP: to MCU/1: 8610BE7E
06:28:21 WIF: Checking connection...
06:28:41 WIF: Checking connection...
06:28:58 TCP: to MCU/1: 665721575485EA4DA66BB5DA55AAC820FB0C42CEACA20BE29CA800ECCC47FB2C7E
06:28:58 TCP: from MCU: 6757A157548515
06:28:58 TCP: from MCU: 63917E

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [forum](https://groups.google.com/d/forum/sonoffusers)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [ ] Self-compiled
    - [ ] IDE / Compiler used: _____
- [ ] Flashing tools used: _____

- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:

06:07:13 CMD: Backlog Template; Module; GPIO 255
06:07:13 SRC: WebConsole from 192.168.1.42
06:07:13 CMD: Group 0, Index 1, Command "BACKLOG", Data "Template; Module; GPIO 255"
06:07:13 SRC: Backlog
06:07:13 CMD: Group 0, Index 1, Command "TEMPLATE", Data ""
06:07:13 RSL: stat/tasmota_DBF93B/RESULT = {"NAME":"Sonoff ZHABridge","GPIO":[56,208,0,209,59,58,0,0,0,0,0,0,17],"FLAG":0,"BASE":18}
06:07:13 SRC: Backlog
06:07:13 CMD: Group 0, Index 1, Command "MODULE", Data ""
06:07:13 RSL: stat/tasmota_DBF93B/RESULT = {"Module":{"0":"Sonoff ZHABridge"}}
06:07:13 SRC: Backlog
06:07:13 CMD: Group 0, Index 1, Command "GPIO", Data "255"
06:07:13 RSL: stat/tasmota_DBF93B/RESULT = {"GPIO0":{"56":"Led1i"},"GPIO1":{"208":"TCP Tx"},"GPIO2":{"0":"None"},"GPIO3":{"209":"TCP Rx"},"GPIO4":{"59":"Led4i"},"GPIO5":{"58":"Led3i"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"0":"None"},"GPIO13":{"0":"None"},"GPIO14":{"0":"None"},"GPIO15":{"0":"None"},"GPIO16":{"17":"Button1"}}
06:07:14 WIF: Checking connection...
```
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```
  Rules output here:

06:33:46 CMD: Backlog Rule1; Rule2; Rule3
06:33:46 SRC: WebConsole from 192.168.1.42
06:33:46 CMD: Group 0, Index 1, Command "BACKLOG", Data "Rule1; Rule2; Rule3"
06:33:46 SRC: Backlog
06:33:46 CMD: Group 0, Index 1, Command "RULE", Data ""
06:33:47 RSL: stat/tasmota_DBF93B/RESULT = {"Rule1":"ON","Once":"OFF","StopOnError":"OFF","Length":37,"Free":474,"Rules":"on system#boot do TCPStart 8888 endon"}
06:33:47 SRC: Backlog
06:33:47 CMD: Group 0, Index 2, Command "RULE", Data ""
06:33:47 RSL: stat/tasmota_DBF93B/RESULT = {"Rule2":"OFF","Once":"OFF","StopOnError":"OFF","Length":0,"Free":511,"Rules":""}
06:33:47 SRC: Backlog
06:33:47 CMD: Group 0, Index 3, Command "RULE", Data ""
06:33:47 RSL: stat/tasmota_DBF93B/RESULT = {"Rule3":"OFF","Once":"OFF","StopOnError":"OFF","Length":0,"Free":511,"Rules":""}

```
- [ ] Provide the output of this command: `Status 0`:
```
  STATUS 0 output here:
06:03:22 CMD: Group 0, Index 1, Command "STATUS", Data "0"
06:03:22 RSL: stat/tasmota_DBF93B/STATUS = {"Status":{"Module":0,"DeviceName":"Tasmota","FriendlyName":["Tasmota"],"Topic":"tasmota_DBF93B","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0}}
06:03:22 RSL: stat/tasmota_DBF93B/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota.bin","RestartReason":"Software/System restart","Uptime":"0T00:12:32","StartupUTC":"2020-09-25T04:50:50","Sleep":50,"CfgHolder":4617,"BootCount":7,"BCResetTime":"2020-09-24T21:22:00","SaveCount":21,"SaveAddress":"F7000"}}
06:03:22 RSL: stat/tasmota_DBF93B/STATUS2 = {"StatusFWR":{"Version":"8.5.0.1(zbbridge)","BuildDateTime":"2020-09-24T18:19:24","Boot":31,"Core":"2_7_4_1","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"348/699"}}
06:03:22 RSL: stat/tasmota_DBF93B/STATUS3 = {"StatusLOG":{"SerialLog":0,"WebLog":4,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["Keenetic-2730",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C8000100060000005A00000000000000","00000000","00006000","00000000"]}}
06:03:22 RSL: stat/tasmota_DBF93B/STATUS4 = {"StatusMEM":{"ProgramSize":526,"Free":476,"Heap":28,"ProgramFlashSize":1024,"FlashSize":2048,"FlashChipId":"1540A1","FlashFrequency":40,"FlashMode":2,"Features":["00000809","0F100786","04500001","00000002","00000000","00000000","00020000","00000000"],"Drivers":"1,2,4,7,9,10,20,21,23,41","Sensors":""}}
06:03:22 RSL: stat/tasmota_DBF93B/STATUS5 = {"StatusNET":{"Hostname":"tasmota_DBF93B-6459","IPAddress":"192.168.1.155","Gateway":"192.168.1.1","Subnetmask":"255.255.255.0","DNSServer":"192.168.1.1","Mac":"A4:CF:12:DB:F9:3B","Webserver":2,"WifiConfig":4,"WifiPower":17.0}}
06:03:22 RSL: stat/tasmota_DBF93B/STATUS6 = {"StatusMQT":{"MqttHost":"","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_DBF93B","MqttUser":"DVES_USER","MqttCount":0,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30}}
06:03:22 RSL: stat/tasmota_DBF93B/STATUS7 = {"StatusTIM":{"UTC":"2020-09-25T05:03:22","Local":"2020-09-25T06:03:22","StartDST":"2020-03-29T02:00:00","EndDST":"2020-10-25T03:00:00","Timezone":"+01:00","Sunrise":"06:41","Sunset":"18:41"}}
06:03:22 RSL: stat/tasmota_DBF93B/STATUS10 = {"StatusSNS":{"Time":"2020-09-25T06:03:22"}}
06:03:22 RSL: stat/tasmota_DBF93B/STATUS11 = {"StatusSTS":{"Time":"2020-09-25T06:03:22","Uptime":"0T00:12:32","UptimeSec":752,"Vcc":3.486,"Heap":28,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Wifi":{"AP":1,"SSId":"Keenetic-2730","BSSId":"E4:18:6B:28:5F:1C","Channel":5,"RSSI":100,"Signal":-43,"LinkCount":1,"Downtime":"0T00:00:03"}}}


```
- [ ] Provide the output of the Console log output when you experience your issue; if applicable:
  _(Please use_ `weblog 4` _for more debug information)_
```
  Console output here:

06:28:14 TCP: to MCU/1: 3350215754211753A97E
06:28:14 TCP: from MCU: 3450A1575421054D
06:28:14 TCP: from MCU: A66BB5DA55AA6DB663B1D854123198C63F7E
06:28:14 TCP: to MCU/1: 8430FC7E
06:28:14 TCP: to MCU/1: 4451215754800487317E
06:28:14 TCP: from MCU: 4551A157548015B5E9954C20AF55384F2C7E
06:28:14 TCP: to MCU/1: 8520DD7E
06:28:14 TCP: to MCU/1: 55562157543D5A7D337E
06:28:14 TCP: from MCU: 5656A157543D86
06:28:14 TCP: from MCU: E3A37E
06:28:14 TCP: to MCU/1: 8610BE7E
06:28:21 WIF: Checking connection...
06:28:41 WIF: Checking connection...
06:28:58 TCP: to MCU/1: 665721575485EA4DA66BB5DA55AAC820FB0C42CEACA20BE29CA800ECCC47FB2C7E
06:28:58 TCP: from MCU: 6757A157548515
06:28:58 TCP: from MCU: 63917E
06:28:58 TCP: to MCU/1: 87009F7E


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

use https://www.digiblur.com/2020/07/how-to-use-sonoff-zigbee-bridge-with.html?m=1 
with latest tasmota-zbbridge.bin  and ncp-uart-sw_6.7.6_115200.ota  or ncp-uart-sw_6.5.5_115200.ota 

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

06:26:37 RSL: tele/tasmota_DBF93B/INFO1 = {"Module":"Sonoff ZbBridge","Version":"8.5.0.1(zbbridge)","FallbackTopic":"cmnd/DVES_DBF93B_fb/","GroupTopic":"cmnd/tasmotas/"}
06:26:37 RSL: tele/tasmota_DBF93B/INFO2 = {"WebServerMode":"Admin","Hostname":"tasmota_DBF93B-6459","IPAddress":"192.168.1.155"}

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment]
Please use no template. Firmware has all settings and functions (per default) which are needed to flash the EFR32 MCU with the ncp firmware. Do a `reset 1` and flash the NCP firmware again. After reset you should see similar (`weblog 4` enabled)

```
16:50:37 ZIG: Resetting EZSP device
16:50:38 MQT: tele/sonoff-064A39/RESULT = {"ZbState":{"Status":1,"Message":"EFR32 booted","RestartReason":"Power-on","Code":2}}
16:50:38 ZIG: ZbEZSPSend 000008
16:50:38 ZIG: {"ZbEZSPReceived":"000008026067"}
16:50:38 MQT: tele/sonoff-064A39/RESULT = {"ZbState":{"Status":55,"Version":"6.7.6.0","Protocol":8,"Stack":2}}
16:50:38 MQT: tele/sonoff-064A39/RESULT = {"ZbState":{"Status":3,"Message":"Configured, starting coordinator"}}
16:50:38 ZIG: ZbEZSPSend 53001E0200
16:50:38 ZIG: {"ZbEZSPReceived":"530000"}
16:50:38 ZIG: ZbEZSPSend 5300061000
16:50:38 ZIG: {"ZbEZSPReceived":"530000"}
16:50:38 ZIG: ZbEZSPSend 53000C0200
16:50:38 ZIG: {"ZbEZSPReceived":"530000"}
16:50:38 ZIG: ZbEZSPSend 53000D0500
16:50:38 ZIG: {"ZbEZSPReceived":"530000"}
16:50:38 ZIG: ZbEZSPSend 5300112000
16:50:38 ZIG: {"ZbEZSPReceived":"530000"}
16:50:38 ZIG: ZbEZSPSend 530012001E
16:50:38 ZIG: {"ZbEZSPReceived":"530000"}
16:50:38 ZIG: ZbEZSPSend 5300190200
16:50:38 ZIG: {"ZbEZSPReceived":"530000"}
16:50:38 ZIG: ZbEZSPSend 53001A1000
16:50:38 ZIG: {"ZbEZSPReceived":"530000"}
16:50:38 ZIG: ZbEZSPSend 53001E0400
16:50:39 ZIG: {"ZbEZSPReceived":"530000"}
16:50:39 ZIG: ZbEZSPSend 5300220200
16:50:39 ZIG: {"ZbEZSPReceived":"530000"}
16:50:39 ZIG: ZbEZSPSend 53002A0300
16:50:39 ZIG: {"ZbEZSPReceived":"530000"}
16:50:39 ZIG: ZbEZSPSend 53002D0100
16:50:39 ZIG: {"ZbEZSPReceived":"530000"}
16:50:39 ZIG: ZbEZSPSend 530001FF00
16:50:39 ZIG: {"ZbEZSPReceived":"000000"}
16:50:39 ZIG: ZbEZSPSend 02000104010500000000
16:50:39 ZIG: {"ZbEZSPReceived":"020000"}
16:50:39 ZIG: ZbEZSPSend 02000B04010500000000
16:50:39 ZIG: {"ZbEZSPReceived":"020000"}
16:50:39 ZIG: ZbEZSPSend 100000F9FF58020807020500
16:50:39 ZIG: {"ZbEZSPReceived":"100000"}
16:50:39 ZIG: ZbEZSPSend 55000003
16:50:39 ZIG: {"ZbEZSPReceived":"550000"}
16:50:39 ZIG: ZbEZSPSend 55000220
16:50:39 ZIG: {"ZbEZSPReceived":"550000"}
16:50:39 ZIG: ZbEZSPSend 55000330
16:50:39 ZIG: {"ZbEZSPReceived":"550000"}
16:50:39 ZIG: ZbEZSPSend 55000551
16:50:39 ZIG: {"ZbEZSPReceived":"550000"}
16:50:39 ZIG: ZbEZSPSend 55000660
16:50:39 ZIG: {"ZbEZSPReceived":"550000"}
16:50:39 ZIG: ZbEZSPSend 17000000
16:50:39 ZIG: {"ZbEZSPReceived":"170093"}
16:50:39 MQT: tele/sonoff-064A39/RESULT = {"ZbState":{"Status":2,"Message":"Resetting configuration"}}
16:50:50 ZIG: Resetting EZSP device
16:50:51 MQT: tele/sonoff-064A39/RESULT = {"ZbState":{"Status":1,"Message":"EFR32 booted","RestartReason":"Power-on","Code":2}}
16:50:51 ZIG: ZbEZSPSend 000008
16:50:51 ZIG: {"ZbEZSPReceived":"000008026067"}
16:50:51 MQT: tele/sonoff-064A39/RESULT = {"ZbState":{"Status":55,"Version":"6.7.6.0","Protocol":8,"Stack":2}}
16:50:51 MQT: tele/sonoff-064A39/RESULT = {"ZbState":{"Status":3,"Message":"Configured, starting coordinator"}}
16:50:51 ZIG: ZbEZSPSend 53001E0200
16:50:51 ZIG: {"ZbEZSPReceived":"530000"}
16:50:51 ZIG: ZbEZSPSend 5300061000
16:50:51 ZIG: {"ZbEZSPReceived":"530000"}
16:50:51 ZIG: ZbEZSPSend 53000C0200
16:50:51 ZIG: {"ZbEZSPReceived":"530000"}
16:50:51 ZIG: ZbEZSPSend 53000D0500
16:50:51 ZIG: {"ZbEZSPReceived":"530000"}
16:50:51 ZIG: ZbEZSPSend 5300112000
16:50:51 ZIG: {"ZbEZSPReceived":"530000"}
16:50:51 ZIG: ZbEZSPSend 530012001E
16:50:51 ZIG: {"ZbEZSPReceived":"530000"}
16:50:51 ZIG: ZbEZSPSend 5300190200
16:50:52 ZIG: {"ZbEZSPReceived":"530000"}
16:50:52 ZIG: ZbEZSPSend 53001A1000
16:50:52 ZIG: {"ZbEZSPReceived":"530000"}
16:50:52 ZIG: ZbEZSPSend 53001E0400
16:50:52 ZIG: {"ZbEZSPReceived":"530000"}
16:50:52 ZIG: ZbEZSPSend 5300220200
16:50:52 ZIG: {"ZbEZSPReceived":"530000"}
16:50:52 ZIG: ZbEZSPSend 53002A0300
16:50:52 ZIG: {"ZbEZSPReceived":"530000"}
16:50:52 ZIG: ZbEZSPSend 53002D0100
16:50:52 ZIG: {"ZbEZSPReceived":"530000"}
16:50:52 ZIG: ZbEZSPSend 530001FF00
16:50:52 ZIG: {"ZbEZSPReceived":"000000"}
16:50:52 ZIG: ZbEZSPSend 02000104010500000000
16:50:52 ZIG: {"ZbEZSPReceived":"020000"}
16:50:52 ZIG: ZbEZSPSend 02000B04010500000000
16:50:52 ZIG: {"ZbEZSPReceived":"020000"}
16:50:52 ZIG: ZbEZSPSend 100000F9FF58020807020500
16:50:52 ZIG: {"ZbEZSPReceived":"100000"}
16:50:52 ZIG: ZbEZSPSend 55000003
16:50:52 ZIG: {"ZbEZSPReceived":"550000"}
16:50:52 ZIG: ZbEZSPSend 55000220
16:50:52 ZIG: {"ZbEZSPReceived":"550000"}
16:50:52 ZIG: ZbEZSPSend 55000330
16:50:52 ZIG: {"ZbEZSPReceived":"550000"}
16:50:52 ZIG: ZbEZSPSend 55000551
16:50:52 ZIG: {"ZbEZSPReceived":"550000"}
16:50:52 ZIG: ZbEZSPSend 55000660
16:50:52 ZIG: {"ZbEZSPReceived":"550000"}
16:50:52 ZIG: ZbEZSPSend 68000C035A6967426565416C6C69616E63653039A1403D4AC82B9606A1403D4AC82B9606000000000000000000
16:50:52 ZIG: {"ZbEZSPReceived":"680000"}
16:50:52 ZIG: ZbEZSPSend 1E00C82B9606CCCCCCCCC82B140B00FFFF0000000000
16:50:53 ZIG: {"ZbEZSPReceived":"1E0000"}
16:50:53 ZIG: {"ZbEZSPReceived":"190090"}
16:50:53 ZIG: ZbEZSPSend 2600
16:50:53 ZIG: {"ZbEZSPReceived":"2600D4A642FEFF23A460"}
16:50:53 ZIG: ZbEZSPSend 2700
16:50:53 ZIG: {"ZbEZSPReceived":"27000000"}
16:50:53 ZIG: Subscribe to group 0 'ZbListen0 0'
16:50:53 ZIG: ZbEZSPSend 64000000000100
16:50:53 ZIG: {"ZbEZSPReceived":"640000"}
16:50:53 MQT: tele/sonoff-064A39/RESULT = {"ZbState":{"Status":0,"Message":"Started"}}
16:50:53 ZIG: Zigbee started
16:50:53 ZIG: Zigbee signature in Flash: 3167697A - 59
16:50:53 ZIG: Zigbee devices data in Flash (59 bytes)
```

If zigbee started appears the NCP firmware is working.

**Comment by abalashov78** [comment]
> 
> 
> Please use no template. Firmware has all settings and functions (per default) which are needed to flash the EFR32 MCU with the ncp firmware. Do a `reset 1` and flash the NCP firmware again. After reset you should see similar (`weblog 4` enabled)
> 

> If zigbee started appears the NCP firmware is working.

Thank you for answer. 

After resetting device i still can't pair my snzb-02 with zbbrudge

Resetting EZSP device
07:41:00 RSL: tele/tasmota_DBF93B/RESULT = {"ZbState":{"Status":1,"Message":"EFR32 booted","RestartReason":"Power-on","Code":2}}
07:41:00 RSL: tele/tasmota_DBF93B/RESULT = {"ZbState":{"Status":55,"Version":"6.7.6.0","Protocol":8,"Stack":2}}
07:41:00 RSL: tele/tasmota_DBF93B/RESULT = {"ZbState":{"Status":3,"Message":"Configured, starting coordinator"}}
07:41:02 ZIG: {"ZbEZSPReceived":"1E0000"}
07:41:02 ZIG: Subscribe to group 0 'ZbListen0 0'
07:41:02 RSL: tele/tasmota_DBF93B/RESULT = {"ZbState":{"Status":0,"Message":"Started"}}
07:41:02 ZIG: Zigbee started
07:41:02 ZIG: No zigbee devices data in Flash
07:41:42 CMD: ZbPermitJoin 1
07:41:42 RSL: tele/tasmota_DBF93B/RESULT = {"ZbState":{"Status":21,"Message":"Pairing mode enabled"}}
07:41:42 RSL: stat/tasmota_DBF93B/RESULT = {"ZbPermitJoin":"Done"}
07:41:42 ZIG: Internal ZDO message 0x0036 sent from 0x0000 (broadcast)

Here i push button on my device for pairing

07:42:42 RSL: tele/tasmota_DBF93B/RESULT = {"ZbState":{"Status":20,"Message":"Pairing mode disabled"}}
07:42:59 CMD: ZbConfig 
07:42:59 RSL: stat/tasmota_DBF93B/RESULT = {"ZbConfig":{"Channel":11,"PanID":"0x0FA4","ExtPanID":"0xCCCCCCCCDB12CFA4","KeyL":"0xDB12CFA4F93F40A1","KeyH":"0xDB12CFA4F93F40A1","TxRadio":20}}

and i have no pairing device

**Comment by abalashov78** [comment]
It's magic
Few minutes later I try pairing device again and it's work!
Thank you so mutch! 

Using no template is correct colution for my problem.

**Comment by abalashov78** [comment]
But i still have problem with ZHA's integration in HA.
Next question is not clear for me. 

Zbbridge is work fine with ncp-uart-sw_6.7.6_115200.ota  and i pair device. 
But with ncp-uart-sw_6.5.5_115200.ota in Home Assistant i can't pair it.


2020-09-25 10:19:30 DEBUG (bellows.thread_0) [bellows.uart] Sending: b'477121575408290d457e'
2020-09-25 10:19:30 DEBUG (bellows.thread_0) [bellows.uart] Data frame: b'7571a157540815e1247e'
2020-09-25 10:19:30 DEBUG (bellows.thread_0) [bellows.uart] Sending: b'8070787e'
2020-09-25 10:19:30 DEBUG (MainThread) [bellows.ezsp.protocol] Application frame 34 (permitJoining) received: b'00'
2020-09-25 10:19:39 DEBUG (MainThread) [bellows.ezsp.protocol] Send command nop: ()
2020-09-25 10:19:39 DEBUG (bellows.thread_0) [bellows.uart] Sending: b'50762157542f23d57e'
2020-09-25 10:19:39 DEBUG (bellows.thread_0) [bellows.uart] Data frame: b'0676a157542f03987e'
2020-09-25 10:19:39 DEBUG (bellows.thread_0) [bellows.uart] Sending: b'8160597e'
2020-09-25 10:19:39 DEBUG (MainThread) [bellows.ezsp.protocol] Application frame 5 (nop) received: b''
2020-09-25 10:19:49 DEBUG (MainThread) [bellows.ezsp.protocol] Send command nop: ()
2020-09-25 10:19:49 DEBUG (bellows.thread_0) [bellows.uart] Sending: b'61772157542fe3a87e'
2020-09-25 10:19:49 DEBUG (bellows.thread_0) [bellows.uart] Data frame: b'1777a157542ff6ed7e'
2020-09-25 10:19:49 DEBUG (bellows.thread_0) [bellows.uart] Sending: b'82503a7e'
2020-09-25 10:19:49 DEBUG (MainThread) [bellows.ezsp.protocol] Application frame 5 (nop) received: b''

https://www.digiblur.com/2020/07/how-to-use-sonoff-zigbee-bridge-with.html

backlog rule1 on system#boot do TCPStart 8888 endon ; rule1 1 ; template {"NAME":"Sonoff ZHABridge","GPIO":[56,208,0,209,59,58,0,0,0,0,0,0,17],"FLAG":0,"BASE":18} ; module 0

if i don't use template - i can't start tcp server on 8888 for connect HA to bridge.

**Comment by s-hadinger** [comment]
@abalashov78 It looks like the issue is now on HA side. I'm afraid we can't help here. I suggest you post the issue in their Github site.

**Comment by abalashov78** [comment]
correct template for latest versions of HA is:

backlog rule1 on system#boot do TCPStart 8888 endon ; rule1 1 ; template {"NAME":"ZHA ZBBridge","GPIO":[56,208,0,209,59,58,0,0,0,0,0,0,17],"FLAG":0,"BASE":18} ; module 0

**Comment by ascillato2** [comment]
Closing this issue as it has been answered.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Community](https://groups.google.com/d/forum/sonoffusers) for forum.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)

**Comment by lehighkid** [comment]
I know this is closed but did want to share that I had the same exact issue following the same guide.  I was not able to pair any devices either via Tasmota or HA.  I was concerned my hardware was damaged after trying different firmware versions, repeating the process verbatim.  Eventually I found it wasn't after finding this thread.  

I started over completely - refreshed Tasmota via serial (likely not necessary) - did not use a template, flashed the radio firmware and then used this template above.  

```backlog rule1 on system#boot do TCPStart 8888 endon ; rule1 1 ; template {"NAME":"ZHA ZBBridge","GPIO":[56,208,0,209,59,58,0,0,0,0,0,0,17],"FLAG":0,"BASE":18} ; module 0```

```Weblog 4``` was very helpful as well to confirm ZB radio function

All is working well now via HA - much, much better than my HUSBZ1 stick in terms of range, responsiveness and so far stickiness.  Thank you.
